test/bats/test.bats

```diff
@@ -398,3 +368,46 @@ EOF
     run kubectl run demo1 --namespace default --image=registry:5000/notation:unsigned
     assert_failure
 }
+
+@test "validate image signed by leaf cert" {
```

Comment:
Do you think we should add same test for azure? Might not be necessary since functionally it should be the same...

Should we add a similar test for the cli test?

Reply:
Added both tests.